src/Microsoft.Net.Http.Headers/MediaTypeHeaderValue.cs

```diff
@@ -59,7 +59,7 @@ public string Charset
                     // Remove charset parameter
                     if (charsetParameter != null)
                     {
-                        _parameters.Remove(charsetParameter);
+                        Parameters.Remove(charsetParameter);
```

Comment:
Slight inconsistency doesn't matter that much. Impossible for this line to be reached if `_parameters` is `null` and the potential allocation at [line 73](https://github.com/aspnet/HttpAbstractions/pull/465/files#diff-afd3378d76454b61ea0cdd22e56685fdR73) is required. I somewhat prefer @rynowak's reduction in `_parameters` use.